### PR TITLE
admin_ceph_upgrade.xml: rewrite MON upgrade warning

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -273,14 +273,17 @@
       </para>
      </step>
     </substeps>
-    <warning>
-     <title>Do not Upgrade/Restart the Third &mon;</title>
+    <note>
+     <title>Regarding &mon; upgrade</title>
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml]:
```suggestion
     <title>Regarding &mon; Upgrade</title>
```

Comment (tbazant) [inline xml/admin_ceph_upgrade.xml]:
```suggestion
     <title>Regarding &mon; Upgrade</title>
```

File: xml/admin_ceph_upgrade.xml [hunk 2]
```diff
@@ -273,14 +273,17 @@
...
      <para>
-      Do not reboot or restart the third &mon; before it is upgraded&mdash;the
-      third &mon; will <emphasis role="bold">not</emphasis> join quorum anymore
-      in case the other two formed Nautilus quorum.
+      Due to a limitation in the &mon; design, once two MONs have been upgraded
+      to &productname; &productnumber; and have formed a quorum, the third MON
+      (while still on &productname; &prevproductnumber;) will not rejoin the
+      MON cluster if it restarted for any reason, including a node reboot.
+      Therefore, once two MONs have been upgraded it is best to upgrade the
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml]:
```suggestion
      Therefore, when two MONs have been upgraded it is best to upgrade the
```

Comment (smithfarm) [reply xml/admin_ceph_upgrade.xml]: Actually, "once" is better in this case ;-)

File: xml/admin_ceph_upgrade.xml (hunk 1 above)
Comment (smithfarm) [reply xml/admin_ceph_upgrade.xml]: The idea was to have the heading be something like "Note Regarding Ceph Monitor Upgrade", but I'll settle for "Note: Regarding Ceph Monitor Upgrade" ;-)

**Pull request merged**